base_wamas/models/wamas_document_default_field.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class WamasDocumentDefaultField(models.Model):
```

Comment:
Please, add docstrings on all classes and methods to explain what they represent/do